.ansible-lint

```diff
@@ -8,23 +9,22 @@ exclude_paths:
   - .yamllint
   - molecule/
   - tests/azure/
-
+  - meta/runtime.yml
+ 
+kinds:
+  - playbook: '**/tests/**/test_*.yml'
+  - playbook: '**/playbooks/**/*.yml'
 
 parseable: true
 
-quiet: false
+quiet: yes
 
 skip_list:
-  - '201'  # Trailing whitespace
-  - '204'  # Lines should be no longer than 160 chars
-  - '206'  # Variables should have spaces before and after: {{ var_name }}'
-  - '208'  # File permissions not mentioned
   - '301'  # Commands should not change things if nothing needs doing'
-  - '305'  # Use shell only when shell functionality is required'
-  - '306'  # Shells that use pipes should set the pipefail option'
-  - '502'  # All tasks should be named
-  - '505'  # Referenced missing file
-
+  - '305'  # Use shell only when shell functionality is required
+  - '306'  # risky-shell-pipe
+  - yaml   # yamllint should be executed separately.
+  - experimental  # Ignore rules that resulted in false positives
```

Comment:
What is `experimental` exactly?

Reply:
`experimental` varies from release to release. It used to give lots of false positive, but is much more stable in the latest versions.

I enabled `experimental` and fixed the warnings for `ansible-lint 5.0.7 using ansible 2.9.25`.